dts/bindings/lora/semtech,sx1276.yaml

```diff
@@ -24,3 +24,8 @@ properties:
         Up to six pins that produce service interrupts from the modem.
 
         These signals are normally active-high.
+
+    lora-rtcs:
```

Comment:
property name should be `zephyr,rtc(s)` since this is a driver specific option and not specifically a hardware related thing.

Reply:
It's an RTC used by the LoRa driver. zephyr,rtcs is somehow confusing. But if it's the preferred name, I'll change it! The (s) is needed because the script complains otherwise.